framework/packages/abstract-std/src/objects/gov_type.rs

```diff
@@ -166,6 +217,7 @@ impl<T: AddressLike> std::fmt::Display for GovernanceDetails<T> {
                 governance_type, ..
             } => governance_type.to_owned(),
             GovernanceDetails::Renounced {} => "renounced".to_string(),
+            GovernanceDetails::NFT { .. } => "nft-account".to_string(),
```

Comment:
```suggestion
            GovernanceDetails::NFT { .. } => "nft".to_string(),
```
nit: `-account` here is superfluous